### PR TITLE
Added alt text prop for Image component and example on website

### LINKED ISSUE #1919: Fabric website: Need alt-text for Image component
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes)

Requested priority: (Normal) 

I realize for our image component, we aren't providing alt-text. We should be doing this to represent a best practice. Should be pretty easy for a good win.

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #1919 
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Added an optional alt text prop for the Image component. Also added this prop to the examples on the website to represent best practices.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Image/examples/Image.None.Example.tsx [hunk 1]
```diff
@@ -19,11 +19,11 @@ export class ImageNoneExample extends React.Component<any, any> {
       <div>
         <p>By setting the imageFit property to "none", the image will remain at its natural size, even if the frame is made larger or smaller by setting the width and height props.</p>
         <Label>The image is larger than the frame, so it is cropped to fit. The image is positioned at the upper left of the frame.</Label>
-        <Image  { ...imageProps as any } />
+        <Image  { ...imageProps as any } alt='larger image fit none example' />
         <br />
```
Comment (battletoilet) [inline packages/office-ui-fabric-react/src/components/Image/examples/Image.None.Example.tsx]: Let's be a bit more verbose with these descriptions as i'm not entirely sure that non-sighted user would understand these. Maybe something like 

```
Example implementation of the property imageFit using the none value.
```

File: packages/office-ui-fabric-react/src/components/Image/Image.Props.ts [hunk 2]
```diff
@@ -30,6 +30,11 @@ export interface IImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
   className?: string;
 
   /**
+   * Specifies alternative text for an image.
+   */
+  alt?: string;
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Image/Image.Props.ts]: you probably don't need to specify this prop. It's part of `React.ImgHTMLAttributes<HTMLImageElement>` which this interface is extending (just like src, height, width etc).

Comment (lynamemi) [reply packages/office-ui-fabric-react/src/components/Image/Image.Props.ts]: That makes sense - one thing that specifying the prop gives us is the added documentation encouraging it's use. @Jahnp are there other places on the site we can use to show that design guidance encourages alt text, other than adding it explicitly as a prop? The "Best Practices" section is currently unused - could be an option? https://dev.office.com/fabric#Best Practices

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/Image/Image.Props.ts]: Yeah, best practices for sure, and definitely in the example. In the future it'd be nice to be able to link out to the type definition being extended.

**Pull request merged**